ansible/undercloud/deploy-undercloud.yml

```diff
@@ -0,0 +1,200 @@
+---
+#
+# Playbook deploying and setting up your undercloud for overcloud deployment in scale lab environment
+#
+# Tested against OSP10 and single-nic-vlans config setup with private external network
+#
+
+# Tasks as root
+- hosts: undercloud
+  gather_facts: true
+  vars_files:
+    - vars/main.yml
+  remote_user: root
+  tasks:
+    - name: Disable epel
+      shell: rpm -e epel-release
+      ignore_errors: true
+
+    - name: Get rhos-release
+      get_url:
+        url: "{{rhos_release_rpm}}"
+        dest: /root/rhos-release.rpm
+        validate_certs: false
+
+    - name: Install rhos-release
+      yum: name=/root/rhos-release.rpm
+
+    - name: Setup OSP version to install
+      command: "rhos-release {{rhos_release}} -p {{build}}"
+
+    - name: Install tmux
```

Comment:
Can you also add screen here please?  tmux has performance issues when you run heavy i/o tasks inside it.